aeron-archive/src/main/cpp/CMakeLists.txt

```diff
@@ -111,6 +111,10 @@ target_link_libraries(aeron_archive_client
     PRIVATE aeron_client
     INTERFACE ${CMAKE_THREAD_LIBS_INIT})
 
+if (MSVC)
+    target_compile_options(aeron_archive_client PRIVATE -std:c++17 -Zc:__cplusplus)
```

Comment:
specifying C++17 is too far. Should not be there. And definitely should not be different than top level CMakeLists.

Reply:
SBE depends on this, otherwise compiling error arise

Reply:
C++17 is not required for SBE. It can use it. But it is not required.

Reply:
This is the compiling error, any suggetion
```
[build] ..\..\aeron-archive\src\main\cpp\client\ControlResponseAdapter.cpp(68): error C2051: case 表达式不是常量
[build] ..\..\aeron-archive\src\main\cpp\client\ControlResponseAdapter.cpp(85): error C2051: case 表达式不是常量
[build] ..\..\aeron-archive\src\main\cpp\client\ControlResponseAdapter.cpp(115): warning C4065: switch 语句包含“default”但是未包含“case”标签
```